client/me/purchases/controller.jsx

```diff
@@ -35,7 +35,7 @@ import titles from './titles';
 import VatInfoPage from './vat-info';
 import useVatDetails from './vat-info/use-vat-details';
 
-const useDataViewPurchasesList = config.isEnabled( 'purchases/purchase-list-dataview' );
+const useDataViewPurchasesList = ! config.isEnabled( 'purchases/purchase-list-dataview' );
```

Comment:
@heydemoura did we intend to commit this change? Now I'm seeing my purchases in a dataview. That work is still behind a feature flag as far as I can tell:

![image](https://github.com/user-attachments/assets/b419b3bc-3bbb-4b70-abe8-87f8df43860c)

Reply:
It wasn't supposed to be committed. I have reverted this patch. Thanks for the follow up.